### PR TITLE
Update osdctl to go1.20.x

### PR DESCRIPTION
**Pull request by clcollins**
Update OSDCTL to go 1.20.x

### CODE REVIEW

File: .github/workflows/create_release.yml [hunk 1]
```diff
@@ -23,7 +23,7 @@ jobs:
       - name: Set up Go
         uses: actions/setup-go@v3.5.0
         with:
-          go-version: 1.19.5
+          go-version: 1.20.5
```
Comment (wking) [inline .github/workflows/create_release.yml:26]: https://go.dev/dl/ suggests 1.20.7 is the current tip.  Does the patch version matter here?  If not, maybe use 0 and explain in the commit message?  If it does, maybe use 1.20.7?

Comment (clcollins) [reply .github/workflows/create_release.yml:26]: This was the version I was using locally.  I'll re-do this with latest.

**Comment by clcollins** [comment]
Test failing awaiting image update, I believe.  @mjlshen  is looking at this.

**Comment by mjlshen** [comment]
Blocked by a successful merge of https://github.com/openshift/release/pull/42755 - with current understanding, this will not work until next week Monday

**Comment by mjlshen** [comment]
Blocked by openshift/release#42755

**Comment by mjlshen** [comment]
with current understanding, this will not work until next week Monday Sept 11th

**Review by mjlshen — APPROVED** [review]
/lgtm

**Comment by mjlshen** [comment]
Closes OSD-18253

**Pull request merged**